vitess.io/_includes/left-nav-menu.html

```diff
@@ -38,12 +38,18 @@ <h4 class="arrow-r">Reference Guides</h4>
         <li class="submenu">
           <h4 class="arrow-r">Other Resources</h4>
           <ul style="display: none">
-            <li><a href="/resources/presentations.html">Presentations</a>
+            <li><a href="/resources/presentations.html">Presentations</a></li>
+            <li style="padding-bottom: 0"><a href="/posts/">Blog</a>
+              <ul style="display: block">
+                <li><a href="/cloud-native-mysql-sharding/">Cloud Native MySQL Sharding</a></li>
+                <li><a href="/scaling-mysql-in-cloud-with-vitess-and-kubernetes/">Scaling MySQL in the Cloud</a></li>
```

Comment:
I agree that it won't scale. But I need to make a change to the JS to get the "Blog" link to at least highlight in the side menu if you're on a blog post since they're different pages. Otherwise, once you link to a blog post page, the left nav doesn't give any indication of where you are. (Similarly, I'd like the TOC area to list other posts when you're looking at a blog post page.)

Anyway, I was hoping to save those changes for a later revision. This will at least scale for one or two more blog posts, which should at least buy us a few weeks.